cluster/gce/util.sh

```diff
@@ -1420,6 +1422,7 @@ ENABLE_WORKLOADCONTROLLER: $(yaml-quote ${ENABLE_WORKLOADCONTROLLER})
 ENABLE_KUBESCHEDULER: $(yaml-quote ${ENABLE_KUBESCHEDULER})
 ENABLE_KUBECONTROLLER: $(yaml-quote ${ENABLE_KUBECONTROLLER})
 ENABLE_ETCD: $(yaml-quote ${ENABLE_ETCD})
+ENABLE_APISERVER_INSECURE_PORT: $(yaml-quote ${ENABLE_APISERVER_INSECURE_PORT})
```

Comment:
Please remove this. I added it in scaleout branch and has been merged in master and is there [above](https://github.com/CentaurusInfra/arktos/pull/952/files#diff-09816a4dd2fd3cf28f97bf9f34330a66336c631830b103d0c96d71eb632dab0dR1420)

Reply:
this has been removed.